Ruqoyah/SELA
# #157424604 Setup mongodb and seed projects to the database

#### What does this PR do?
This PR sets up mongodb and also send projects to the database.

#### Description of Task to be completed?
This task sets up mongodb and also send projects to the database.

#### How should this be manually tested?
- Clone the repo
- Run `npm install`
- Navigate to `http://localhost:8000/api/projects/seed` using postman, you should be able to see a message `Projects seeded successfully`

#### What are the relevant pivotal tracker stories?
[157424604](https://www.pivotaltracker.com/story/show/157424604)

#### Any background context you want to add?
N/A

#### Screenshots
N/A
